Week3/homework/step2-7.js

```diff
@@ -20,4 +24,9 @@ f2(y);
 
 console.log(y);
 
-// Add your explanation as a comment here
+/* the value of the object is stored in a reference on memory 
+so when we give a new value to the same object, 
+the new value would be stored on the same reference
+(the same reference of the original object)
+so the original value of the object will change 
+so F2(Y) === Y */
```

Comment:
Looks like you understand the difference, one is copy by value and the ofter is a copy by reference.

Reply:
Yes, that what I meant! 💯 Thanks for making shorter! :) 👍